src/components/ModuleSidebar/moduleSidebar.scss

```diff
@@ -0,0 +1,48 @@
+$sidebar-links-color: #8f5806;
+$link-color: #523304;
```

Comment:
The variable `$link-color` assumes a font color for a link only. Meanwhile, it's been used for the background. Can we use a more descriptive name such as `$brown-color` or whatever?